web/src/features/charts/MissingExchangeData.tsx

```diff
@@ -0,0 +1,46 @@
+import { useTranslation } from 'react-i18next';
+
+import { useNetExchangeChartData } from './hooks/useNetExchangeChartData';
+import { AreaGraphElement } from './types';
+
+const SIGNIFICANT_THRESHOLD = 0.2;
+
+const getTotal = (object: object) => Object.keys(object).length;
+const getNulls = (object: object) =>
+  Object.values(object).filter((v) => v === null).length;
```

Comment:
Hm, so my understanding is that null is the value that represents that we haven't received the data yet, unlike NaN and Infinity. I think we would probably want the application to error on Nan and Infinity, since that's probably an indication that we have a bug. What do you think?